cmd/esc/cli/testdata/env-set-string.yaml

```diff
@@ -0,0 +1,14 @@
+run: |
+  esc env init default/test
+  esc env set default/test path "[a" --string
+  esc env get default/test path --value string
```

Comment:
Oh--nit that we should grab the whole env so that we can see the exact way this gets serialized